src/node_crypto.cc

```diff
@@ -757,6 +757,10 @@ void SecureContext::SetDHParam(const FunctionCallbackInfo<Value>& args) {
   if (dh == nullptr)
     return;
 
+  if (BN_num_bits(dh->p) < 1024) {
+    return env->ThrowError("DH parameter is less than 1024bits");
+  }
```

Comment:
Can you drop the braces?  It's a little inconsistent with the style used in the surrounding code.